src/decimal/src/uint.rs

```diff
@@ -16,6 +16,9 @@ construct_uint! {
 construct_uint! {
     pub struct U192(3);
```

Comment:
Also remove all unused `uint ` type add converting functions from decimal. You do not need functions such as `checked_u320_to_u256` since we've already have generic solution